src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -377,14 +399,19 @@ public EmitOptions WithIncludePrivateMembers(bool value)
             return new EmitOptions(this) { IncludePrivateMembers = value };
         }
 
-        public EmitOptions WithInstrument(string instrument)
+        public EmitOptions WithInstrument(params InstrumentationKind[] instrument)
```

Comment:
>WithInstrument [](start = 27, length = 14)

Please remove this from the API. We generally have only the single overload with an immutable input for the With methods.